include/gadget/types.h

```diff
@@ -34,4 +34,7 @@ typedef __u64 gadget_mntns_id;
 // time.
 typedef __u64 gadget_timestamp;
 
+// gadget_signal is used to represent a unix signal. It's automatically converted to the signal name.
+typedef __u32 gadget_signal;
```

Comment:
nit: Maybe "a field is automatically added that contains the name as string" or something like it.